### PR TITLE
[Student][MBL-13077,MBL-13078] : Fixed some problems with nightly tests

### PR DESCRIPTION
**Pull request by joehoag**
MBL-13077: Fixed low-res scrolling issues in AssignmentsE2ETest
MBL-13078: Fixed a mock canvas conversations API glitch.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: libs/canvas-api-2/src/main/java/com/instructure/canvasapi2/apis/InboxApi.kt [hunk 1]
```diff
@@ -46,7 +46,7 @@ object InboxApi {
 
     internal interface InboxInterface {
 
-        @GET("conversations/?interleave_submissions=1&include[]=participant_avatars")
+        @GET("conversations?interleave_submissions=1&include[]=participant_avatars")
```
Comment (joehoag) [inline libs/canvas-api-2/src/main/java/com/instructure/canvasapi2/apis/InboxApi.kt]: The (presumably unnecessary?) extra "/" caused our MockCanvas logic to not parse the URL correctly, which caused the call to stall, which caused the test to hang/fail on API 23.

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentDetailsPage.kt [hunk 2]
```diff
@@ -17,10 +17,12 @@
 package com.instructure.student.ui.pages
```
Comment (joehoag) [inline apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentDetailsPage.kt]: In general, these changes allow us to scroll up and down to find views when we are running on a low-res device.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1003849&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by joehoag** [comment]
Looks like my slight tweak to the conversations API call messed up the ditto functionality of that call.   I'll regenerate the teacher tapes.

File: libs/canvas-api-2/src/main/java/com/instructure/canvasapi2/apis/InboxApi.kt (hunk 1 above)
Comment (StephenBrough) [reply libs/canvas-api-2/src/main/java/com/instructure/canvasapi2/apis/InboxApi.kt]: Ooh, good catch!

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentDetailsPage.kt (hunk 2 above)
Comment (StephenBrough) [reply apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentDetailsPage.kt]: _Classic low-res_

**Pull request merged**